cmd/nodeproblemdetector/node_problem_detector_test.go

```diff
@@ -79,13 +79,11 @@ func (ne *nullExporter) ExportProblems(*types.Status) {
 
 func TestNPDMain(t *testing.T) {
 	npdo, cleanup := setupNPD(t)
-	defer cleanup()
+	cleanup()
```

Comment:
Missing a defer here.

Reply:
It turned out that the original code will put error into channel to simulate timeout. And will cal cancel function to do the same thing.

Reply:
fixed.Thanks!